src/lang/translation_provider_FILE.cpp

```diff
@@ -0,0 +1,6 @@
+
+#include "translation_provider_FILE.hpp"
+
+//path cannot be longer than 16 characters
+FILEtranslationProvider fileProviderInternal("/internal/ts.mo");
+FILEtranslationProvider fileProviderUSB("/usb/lang/ts.mo");
```

Comment:
good enough for this prototype implementation, we'll think about the location of these providers when we'll have more of them